backend/src/chats/chat/chat.controller.ts

```diff
@@ -59,6 +120,13 @@ export class ChatController {
 		});
 	}
 
+	@Get(':id/messages')
+	async chatMessages(@Param('id') id: number) {
```

Comment:
is this something we use? or maybe if we join a channel later, and we would like to see all the current messages? and prob also with general channels, in which we can only look at the messages?

Reply:
1. yes it is used, it gets the messages from a specific chat. you can find it in `getActiveChatMessages` action from `messageStore`
2. it doesn't matter when you join a channel, it's just for getting the messages from any chat to the frontend
3. not sure what you mean with the last question :)

Reply:
cool